### DISCUSSION TITLE
Welcome to ListExtender Discussions!

### DISCUSSION THREAD

**Discussion by JLambertazzo** [JLambertazzo/ListExtender#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by judy-n** [comment]
Hi! My name is Judy and I've contributed a bit to the repo before (styling/simple features). I'm looking forward to seeing what people contribute to ListExtender 😸

**Reply by JLambertazzo** [reply]
Hi Judy! 👋
